### DISCUSSION TITLE
WL_Customization What parameters must be passed?

### DISCUSSION THREAD

**Discussion by muzafferjoya** [Sunbird-Ed/Community#717] [Installation]
https://github.com/Sunbird-Ed/SunbirdEd-portal/blob/release-6.0.0_RC39/Jenkinsfile

**Comment by gandham-santhosh** [comment]
@muzafferjoya You dont have to pass any params to this. This param was used to build the custom dependencies like resource bundles which no longer in use

**Comment by anisha147** [comment]
Hi @muzafferjoya 

We hope that your queries are resolved. If yes, could you please close this thread from your end?

Thanks!

**Comment by HarishGangula** [comment]
As we haven't received a response from you, we will proceed to close this thread. If you have any further questions or require assistance in the future, please don't hesitate to start a new thread. Thank you.
